apps/mobile/src/app/wallet/browser.tsx

```diff
@@ -0,0 +1,212 @@
+import { useRef, useState } from 'react';
```

Comment:
Maybe this file could be split up into 3? `BrowerInUse`, `BrowserEmptyState` + `BrowserScreen`?